index.js

```diff
@@ -21,7 +21,7 @@ module.exports = function transformSvelte(file, options) {
 			const name = toPascalCase(base.replace(extension, ``))
 
 			try {
-				const svelteOptions = Object.assign({}, options.svelte, {
+				const {_, ...svelteOptions} = Object.assign({}, options.svelte, {
```

Comment:
What's the spread for?  Is there an underscore `_` property that's being removed by this?